xbmc/AppParamParser.h

```diff
@@ -8,22 +8,30 @@
 
 #pragma once
 
+#include <string>
```

Comment:
Include order is own headers before C(++) and system headers, see https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#6-headers

Reply:
sure, thanks